### DISCUSSION TITLE
Δήλωση πρόθεσης της προαιρετικής παρουσίασης

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#68] [Show and tell]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης ερευνητικής εργασίας με την ανάρτηση τουλάχιστον μια μέρα πριν την συμπληρωματική ΟΣΣ

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη συμπληρωματική ΟΣΣ

**Comment by afoscolou** [comment]
Θα ήθελα να ρωτήσω αν θα πραγματοποιηθεί η συμπληρωματική ΟΣΣ που έχει προγραμματιστεί για αύριο Σάββατο 4/3 δεδομένου ότι δεν έχει δηλωθεί καμία πρόθεση προαιρετικής παρουσίασης ως τώρα. Υπήρξε βεβαρημένος φόρτος εργασίας μέχρι τώρα σε αρκετούς από εμάς λόγω υποχρεώσεων στις άλλες θεματικές (ΣΔΥ61 και ΣΔΥ62) και αυτός νομίζω ότι είναι ένας από τους κύριους λόγους αυτής της "αποχής". Προσωπικά δηλώνω την πρόθεσή μου να παρουσιάσω κατά την επόμενη συμπληρωματική ΟΣΣ που θα μπορούσαμε να κάνουμε π.χ. το επόμενο Σάββατο 11/3, αν υπάρχει συμφωνία και από εσάς.

**Reply by epidrome** [reply]
θα ήταν καλό να έχουμε 1-2 παρουσιάσεις και από την πλευρά σας, οπότε ας το αναβάλλουμε για τις 11 Μαρτίου #67

**Reply by sotkot** [reply]
Προτίθεμαι να παρουσιάσω κι εγώ στις 11/03 που θα λάβει χώρα η ΟΣΣ.

**Comment by afoscolou** [comment]
Επισυνάπτω το αρχείο pdf της παρουσίασης που θα κάνω στην αυριανή ΟΣΣ (Σάββατο 11/3 @6μμ) :

1. Shehade, M., & Stylianou-Lambert, T. (2020). Virtual reality in museums: Exploring the experiences of museum professionals. *Applied sciences, 10*(11), 4031.

2. DOI: https://doi.org/10.3390/app10114031

3. [SDY60_4ERG_FOSCOLOU_A_Part D.pdf](https://github.com/sdy60/syllabus/files/10946981/SDY60_4ERG_FOSCOLOU_A_Part.D.pdf)

**Comment by sotkot** [comment]
Καλησπέρα σας,

Ακολουθούν τα σχετικά στοιχεία για την αυριανή μου παρουσίαση (11/3 - 18:00):

1. Roberto Andreoli, Angela Corolla, Armando Faggiano, Delfina Malandrino, Donato Pirozzi, Mirta Ranaldi, Gianluca
Santangelo, and Vittorio Scarano. 2017. A Framework to Design, Develop, and Evaluate Immersive and Collaborative
Serious Games in Cultural Heritage. ACM J. Comput. Cult. Herit. 11, 1, Article 4 (December 2017), 22 pages.

2. DOI: https://doi.org/10.1145/3064644

3. [SDY60_4ERG_PART_D_KOTROTSIOS_SOTIRIOS.pdf](https://github.com/sdy60/syllabus/files/10944002/SDY60_4ERG_PART_D_KOTROTSIOS_SOTIRIOS.pdf)

**Comment by gregtsag** [comment]
θα παρουσιάσω κι εγώ αύριο 11/3

https://doi.org/10.1145/3196709.3196714

**Reply by gregtsag** [reply]
επειδή προέκυψε κάτι έκτακτο δεν θα μπορέσω να είμαι στην σημερινή ΟΣΣ, θα ήθελα αν γίνεται να παρουσιάσω σε επόμενη. ευχαριστώ για την κατανόηση

**Comment by NikosAndreadakis** [comment]
θα γίνει παρουσίαση στο άρθρο:
Haptic virtual reality and immersive learning for enhanced organic chemistry instruction 
Bosede Iyiade Edwards1,2 · Kevin S. Bielawski1,2 · Rui Prada3 · Adrian David Cheok1,2

[DOI](https://doi.org/10.1007/s10055-018-0345-4)

[SDY60_4ERG_ANDREADAKHS_NIKOLAOS_PART_D.pdf](https://github.com/sdy60/syllabus/files/10946772/default.pdf)

**Comment by ghost** [comment]
Παρουσίαση άρθρου για το προαιρετικό μέρος Δ της εργασίας
[SDY60_4ERG_PART_D_KAMPOUROPOULOS.pdf](https://github.com/sdy60/syllabus/files/10947986/SDY60_4ERG_PART_D_KAMPOUROPOULOS.pdf)
https://doi.org/10.1016/j.cag.2017.09.001
Gimeno, J., Portalés, C., Coma, I., Fernández, M., & Martínez, B. (2017). Combining traditional and indirect augmented reality for indoor crowded environments. A case study on the Casa Batlló museum. Computers & graphics, 69, 92-103.

**Comment by vkaragia** [comment]
Επισυνάπτω τα αρχεία pdf της παρουσίασης και τα speaking notes που θα κάνω στην αυριανή ΟΣΣ (Κυριακή 26/3) :

Svevo tour: The design and the experimentation of an augmented reality application for engaging visitors of a literary museum

DOI: https://doi.org/10.1016/j.ijhcs.2018.01.009
[ SDY60_4ERG_PART_D_KARAGIANNAKIS_VASILEIOS_SPEAKING NOTES.docx](https://github.com/sdy60/syllabus/files/11070298/SDY60_4ERG_PART_D_KARAGIANNAKIS_VASILEIOS_SPEAKING.NOTES.docx)
[SDY60_4ERG_PART_D_KARAGIANNAKIS_VASILEIOS.pdf](https://github.com/sdy60/syllabus/files/11070308/SDY60_4ERG_PART_D_KARAGIANNAKIS_VASILEIOS.pdf)
